cmd/did-method-cli/internal/configcommon/helpers_test.go

```diff
@@ -0,0 +1,207 @@
+		require.NoError(t, err)
+
+		_, err = file.WriteString("aooga")
+		require.NoError(t, err)
+
+		defer func() { require.NoError(t, os.Remove(file.Name())) }()
+
+		require.NoError(t, os.Setenv(ConfigFileEnvKey, file.Name()))
+
+		_, err = GetConfig(&cobra.Command{})
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "failed unmarshal")
+	})
+
+	t.Run("fail: can't read member key file", func(t *testing.T) {
+		os.Clearenv()
+
+		file, err := ioutil.TempFile("", "*.json")
+		require.NoError(t, err)
+
+		_, err = file.WriteString(fmt.Sprintf(configData, "bad.file"))
+		require.NoError(t, err)
+
+		defer func() { require.NoError(t, os.Remove(file.Name())) }()
+
+		require.NoError(t, os.Setenv(ConfigFileEnvKey, file.Name()))
+
+		_, err = GetConfig(&cobra.Command{})
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "failed to read jwk file")
+	})
+
+	t.Run("fail: corrupted member key file", func(t *testing.T) {
+		os.Clearenv()
+
+		jwkFile, err := ioutil.TempFile("", "*.json")
+		require.NoError(t, err)
+
+		defer func() { require.NoError(t, os.Remove(jwkFile.Name())) }()
+
+		_, err = jwkFile.WriteString("bad data bad data")
+		require.NoError(t, err)
+
+		file, err := ioutil.TempFile("", "*.json")
+		require.NoError(t, err)
+
+		_, err = file.WriteString(fmt.Sprintf(configData, jwkFile.Name()))
+		require.NoError(t, err)
+
+		defer func() { require.NoError(t, os.Remove(file.Name())) }()
+
+		require.NoError(t, os.Setenv(ConfigFileEnvKey, file.Name()))
+
+		_, err = GetConfig(&cobra.Command{})
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "failed to unmarshal to jwk")
+	})
+
+	t.Run("test get config from temp file", func(t *testing.T) {
+		os.Clearenv()
+
+		jwkFile, err := ioutil.TempFile("", "*.json")
+		require.NoError(t, err)
+
+		defer func() { require.NoError(t, os.Remove(jwkFile.Name())) }()
+
+		_, err = jwkFile.WriteString(jwkData)
+		require.NoError(t, err)
+
+		file, err := ioutil.TempFile("", "*.json")
+		require.NoError(t, err)
+
+		_, err = file.WriteString(fmt.Sprintf(configData, jwkFile.Name()))
+		require.NoError(t, err)
+
+		defer func() { require.NoError(t, os.Remove(file.Name())) }()
+
+		require.NoError(t, os.Setenv(ConfigFileEnvKey, file.Name()))
+
+		c, err := GetConfig(&cobra.Command{})
+		require.NoError(t, err)
+
+		require.Equal(t, "consortium.net", c.ConsortiumData.Domain)
+		require.Len(t, c.MembersData, 1)
+
+		member := c.MembersData[0]
+		require.NotNil(t, member)
+		require.Equal(t, "stakeholder.one", member.Domain)
+	})
+}
+
+func getKey(t *testing.T, data string) jose.SigningKey {
+	key := jose.JSONWebKey{}
+
+	err := key.UnmarshalJSON([]byte(data))
+	require.NoError(t, err)
+
+	return jose.SigningKey{Key: key.Key, Algorithm: jose.EdDSA}
+}
+
+func TestSignConfig(t *testing.T) {
+	t.Run("success", func(t *testing.T) {
+		key := getKey(t, jwkData)
+
+		_, err := SignConfig([]byte("data data data"), []jose.SigningKey{key})
+		require.NoError(t, err)
```

Comment:
Can you check that config is actually signed? You should not ignore return values in tests.